packages/x-components/src/design-system/components/sliding-panel/default.scss

```diff
@@ -78,25 +84,15 @@
 
     &::before {
       // color
-      background: linear-gradient(
-        to right,
-        var(--x-color-background-sliding-panel) 43%,
-        var(--x-color-background-sliding-panel-middle) 67%,
-        var(--x-color-background-sliding-panel-end)
-      );
+      mask: linear-gradient(to right, black 43%, rgba(0, 0, 0, 0.8) 67%, rgba(0, 0, 0, 0));
```

Comment:
When I was talking with you about this I was thinking in applying the mask to the `__scroll` element, instead of the `before` or `after` elements so we don't have to care about the background anymore or those elements.